app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -47,7 +47,8 @@ class SensorReceiver : BroadcastReceiver() {
             StepsSensorManager(),
             StorageSensorManager(),
             TimeZoneManager(),
-            TrafficStatsManager()
+            TrafficStatsManager(),
+            MobileDataManager()
```

Comment:
Should we consider keeping this list alphabetized?

Reply:
Sure, sorry for that, it is fixed now.